### DISCUSSION TITLE
Write patched boot.img failed

### DISCUSSION THREAD

**Discussion by ballparkme** [bmax121/APatch#634] [Q&A]
This issue is formally addressed, but the related answer has gone. Could anyone leave a solution here?
![Screenshot_20240828-004514804](https://github.com/user-attachments/assets/3390f207-c619-4ef4-b862-417c682ceacf)

**Comment by NikunjK8** [comment]
Same thing here. Looking for solution

![Screenshot_2024-08-30-12-40-42-926_me bmax apatch](https://github.com/user-attachments/assets/93eb23c0-4065-4826-9320-c4cf692b7f9c)
![Screenshot_2024-08-30-12-40-48-031_me bmax apatch](https://github.com/user-attachments/assets/de3925b2-296d-44b4-b03f-e994e41abd5b)


![Screenshot_2024-08-30-12-40-10-725_me bmax apatch](https://github.com/user-attachments/assets/0b2ab06d-d677-4855-b3d3-683c39a21bf3)

**Reply by marat2509** [reply]
Try latest nightly

**Reply by NikunjK8** [reply]
CONFIG_KALLSYMS is disabled in my kernel. Does that mean I'll not be able to use APatch?

**Reply by marat2509** [reply]
Yes, if you don't rebuild the kernel adding KALLSYMS support

**Reply by NikunjK8** [reply]
I see. Thanks for clarifying.

**Comment by marat2509** [comment]
> This issue is formally addressed, but the related answer has gone. Could anyone leave a solution here?

Try latest nightly
